src/material-examples/cdk-datepicker-value/cdk-datepicker-value-example.ts

```diff
@@ -12,27 +12,27 @@ export class CdkDatepickerValueExample {
   date = new FormControl(new Date());
   serializedDate = new FormControl((new Date()).toISOString());
   dates: Date[] = [];
-  messages: string[] = [];
 
   constructor(private _dateAdapter: DateAdapter<Date>) {
     this.dates.push(this._dateAdapter.addCalendarDays(this._dateAdapter.today(), 5));
     this.dates.push(this._dateAdapter.addCalendarDays(this._dateAdapter.today(), 10));
     this.dates.push(this._dateAdapter.addCalendarDays(this._dateAdapter.today(), 15));
   }
-
-  _dateSelected(d: Date) {
-    this.messages.push(`Date has changed to ${d.toString()}`);
-  }
 }
 
 
 @Component({
   selector: 'my-value-calendar',
   outputs: ['selectedChange'],
+  styleUrls: ['cdk-datepicker-value-example.css'],
```

Comment:
this is actually the style sheet for the example component, so I would just inline these styles instead:

```ts
styles: [`
  .calendar {
    ...
  }
`]
```